examples/benchmark/tls_bench.c

```diff
@@ -1633,7 +1653,7 @@ static void* server_thread(void* args)
     }
 
     pthread_cond_signal(&info->to_client.cond);
-    info->to_server.done = 1;
+    info->to_client.done = 1;
```

Comment:
Please revert this line. The way it was previously is valid. My recommend for this fix was not right.